crypto/fipsmodule/cipher/cipher.c

```diff
@@ -439,8 +442,7 @@ int EVP_DecryptFinal_ex(EVP_CIPHER_CTX *ctx, unsigned char *out, int *out_len) {
     } else {
       *out_len = i;
     }
-    AES_verify_service_indicator(ctx, 0);
-    return 1;
+    goto success;
```

Comment:
The destination of goto in EVP_DecryptFinal_ex is always success, which is different from EVP_EncryptFinal_ex, is that intended? Is there nothing that may need locking in the call on l.439?